src/lib/models/suggestMultiSelect/index.ts

```diff
@@ -0,0 +1,4 @@
+export interface Option {
+  value: string;
+  img?: string;
```

Comment:
```suggestion
  leftIcon?: string;
```

Would `leftIcon` be a better name?